templates/emails/base.txt.jinja2

```diff
@@ -0,0 +1,6 @@
+{% block content %} {% endblock %}
+
+Best,
+  SimpleLogin team.
```

Comment:
I think we can remove the space here, so something like:

```
Best,
SimpleLogin team.
```

Reply:
Changed